BUILD.md

```diff
@@ -74,6 +80,9 @@ compiler.
 Artichoke specifically requires clang. WebAssembly targets require clang-8 or
 newer.
 
+On Windows, install the latest LLVM distribution from GitHub and add LLVM to
+your PATH: <https://github.com/llvm/llvm-project/releases>.
+
```

Comment:
I already had llvm when things didn't work, but after your suggestion also downloaded a ruby install, so that might be necessary as well...unless I did something else that helped...

Reply:
Yes, Ruby is required to build. I believe this is outlined in `BUILD.md` a bit further down.